keymanapp/keyman
# chore(common): Add crowdin strings for Italian

This pulls in the crowdin strings for Italian (language tag `it`).

**TODO before testing**
- [x] @sgschantz to run the iOS and macOS projects through XCode to add `it` locale.

## User Testing
Note, the overall crowdin progress on Italian is 98%, so there will still be a few untranslated strings that show in English.

* **TEST_ANDROID**
1. Load the PR build of Keyman for Android
2. From the Keyman "Settings" menu, change the "Display Language" to "Italiana"
3. Return to the Keyman app and verify the strings are in Italian

![Android - italian](https://user-images.githubusercontent.com/7358010/167340211-f1a57109-a544-4982-8bf1-d35ddd662d76.png)

* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Italian
3. Return to Keyman and verify strings are in Italian

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the device locale to Italian
3. Return to Keyman and verify strings are in Italian.
